fsspec/fuse.py

```diff
@@ -112,6 +119,8 @@ def release(self, path, fh):
     def chmod(self, path, mode):
         raise NotImplementedError
 
+    def ioctl(self, path, cmd, arg, fip, flags, data):
+        super().ioctl(path, cmd, arg, fip, flags, data)
```

Comment:
Why is this ever called? I don't think it makes sense to try to do anything "device"-oriented for fsspec implementations. The line of code here, is simply a no-op, though, right?

Reply:
Yes, this is just a shim in place while I diagnose the fuse test error. Will drop, unless it appears necessary for the test pass, before merge.